### PR TITLE
Increase number of phases in GC telemetry

### PR DESCRIPTION
**Pull request by PaulBone**
See https://bugzilla.mozilla.org/show_bug.cgi?id=1555101

Checklist for reviewer:

- [x] Scan the PR and verify that no changes (particularly to `.circleci/config.yml`) will cause environment variables (particularly credentials) to be exposed in test logs
- [x] Trigger the `integration` CI test by pushing this revision [as discussed in the README](https://github.com/mozilla-services/mozilla-pipeline-schemas#packaging-and-integration-tests-optional)

For glean changes:
- [ ] Update `include/glean/CHANGELOG.md`

### CODE REVIEW

File: templates/include/telemetry/gcItem.1.schema.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
         "properties": {
           "times": {
             "type": "object",
-            "maxProperties": 65
+            "maxProperties": 72
```
Comment (acmiyaguchi) [inline templates/include/telemetry/gcItem.1.schema.json]: Why 72 instead of 73 as referenced in the bug?

Comment (PaulBone) [reply templates/include/telemetry/gcItem.1.schema.json]: Oops.

There are 73 phases, numbered 0 to 72.  I'm not sure if phase 0 is used but lets make this 73 and I'll fix the mozilla-central patch and also document it there.

Thanks for catching this.

**Comment by PaulBone** [comment]
@acmiyaguchi Would you mind looking at this again?  Thanks.

**Comment by PaulBone** [comment]
@acmiyaguchi Has this been deployed now? can I land the mozilla-central part?

Thanks.

**Comment by acmiyaguchi** [comment]
@PaulBone Yes, the currently deployed schemas are found under `generated-schemas`. Automation will generally generate and deploy schemas around 5pm PST.

Here's a snippet in the `generated-schemas` branch:
https://github.com/mozilla-services/mozilla-pipeline-schemas/blob/ea2e370ffd08361c3cdd6dc1d86b433a4a50b6f6/schemas/telemetry/main/main.4.schema.json#L1279-L1284

**Comment by PaulBone** [comment]
Thanks.

**Pull request merged**